lib/go/test/service_test.go

```diff
@@ -399,6 +399,21 @@ func TestContracts(t *testing.T) {
 
 	})
 
+	t.Run("Should check if payer has sufficient balance to execute tx", func(t *testing.T) {
+		// TODO: Is it better to set up a mock tx and test against it?
```

Comment:
Can/Should I test it against some mocked tx?

Reply:
yea. I think there should be 2 tests one covering a case when the payer has enough FLOW, and one covering when it doesn't.

It looks like the contract doesn't validate the range of the 2 numeric arguments, so you can probably just use a very large value for `gasLimit` to find one that causes the request to return false.

Reply:
I guess I'm running into https://github.com/onflow/flow-core-contracts/blob/master/contracts/FlowFees.cdc#L111